app/controllers/v1/events_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module V1
+  class EventsController < ApplicationController
+    before_action :set_app
+    before_action :set_event_schema
+
+    #  GET /api/v1/consume-events
+    def consume
+      @event = @event_schema.events.create(
```

Comment:
I think we should use `create!`, So we raise an exception if there is something wrong.

Reply:
Done :heavy_check_mark: